src/main/java/com/google/api/codegen/config/GapicProductConfig.java

```diff
@@ -211,10 +211,21 @@ public static GapicProductConfig create(
 
     DiagCollector diagCollector = model.getDiagReporter().getDiagCollector();
 
-    Map<Resource, ProtoFile> resourceDefs =
+    Map<Resource, ProtoFile> resourceFileLevelDefs =
         protoParser.getResourceDefs(sourceProtos, diagCollector);
     Map<ResourceSet, ProtoFile> resourceSetDefs =
         protoParser.getResourceSetDefs(sourceProtos, diagCollector);
+    LinkedHashMap<Resource, ProtoFile> resourcesDefinedInSetsBuilder = new LinkedHashMap<>();
```

Comment:
fixes a bug where we didn't include Resources that are defined within a parent ResourceSet definition